docs/developer-setup/automated-releases.mdx

```diff
@@ -87,21 +87,22 @@ Where the app icon is an image of the appropriate size and the app name is an XM
 
 As part of integrating Continuous Delivery(CD) into the development lifecycle, CI is set up to generate an APK.
 
-The configuration requires the tag to:
-* The product name: _opensrp_
+The configuration requires the tag to have the following parts/sections in order:
+* The prefix `v`
 * Followed by the _version_ in the format `[0-9]+.[0-9]+.[0-9]+[0-9a-zA-Z.-]+`
+* The APK flavor as set up in the gradle build configuration : _quest_
```

Comment:
let's keep quest replaced w/OpenSRP throughout, we want that to be the external name for the default flavor.

Quest is a module name for the UI portion of the code.

We also want other flavors to be prefixed w/opensrp, like opensrp-myproject-versionnumber.apk

Reply:
Will be updating to this